app/jobs/transfer_email_reply_job.rb

```diff
@@ -7,10 +7,16 @@ class TransferEmailReplyJob < ApplicationJob
   self.log_arguments = false
 
   def self.reply_address_for_rdv(rdv)
-    "rdv+#{rdv.uuid}@reply.rdv-solidarites.fr"
+    return nil if rdv.domain.reply_host_name.nil?
```

Comment:
j’ai fait un test en local, lorsqu’on appelle dans le mailer `default ... reply_to: nil` cela envoie un mail sans reply-to, ce qui est bien ce qu’on veut ici